components/TodoList/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react'
+import { View, Text, TextInput, Button } from 'react-native'
+import { observer, useQuery, $root } from 'startupjs'
+import Todo from '../Todo'
+import './index.styl'
+
+export default observer(function ToDoList () {
+  const [taskText, setTaskText] = useState('')
+  const [todos = [], $todos] = useQuery('todos', {
+    $sort: {
+      createdAt: -1
+    }
+  })
+  if (!todos) return
+  function addTask () {
+    if (!taskText.length) return
+    $todos.add({ text: taskText, isDone: false, createdAt: Date.now() })
```

Comment:
@ice9k please create orm model for todos collection to add a todo to the collection and incapsulate default fields.